extensions/ql-vscode/src/remote-queries/run-remote-query.ts

```diff
@@ -358,18 +360,31 @@ export function parseResponse(owner: string, repo: string, response: QueriesResp
   let logMessage = `Successfully scheduled runs on ${numRepositoriesQueried} repositories. See https://github.com/${owner}/${repo}/actions/runs/${response.workflow_run_id}.`;
   logMessage += `${eol2}Repositories queried:${eol}${repositoriesQueried.join(', ')}`;
   if (response.errors) {
+    const { invalid_repositories, repositories_without_database, private_repositories, cutoff_repositories, cutoff_repositories_count } = response.errors;
     logMessage += `${eol2}Some repositories could not be scheduled.`;
-    if (response.errors.invalid_repositories?.length) {
-      logMessage += `${eol2}Invalid repositories:${eol}${response.errors.invalid_repositories.join(', ')}`;
+    if (invalid_repositories?.length) {
+      logMessage += `${eol2}${invalid_repositories.length} repositories were invalid and could not be found:${eol}${invalid_repositories.join(', ')}`;
     }
-    if (response.errors.repositories_without_database?.length) {
-      logMessage += `${eol2}Repositories without databases:${eol}${response.errors.repositories_without_database.join(', ')}`;
+    if (repositories_without_database?.length) {
+      logMessage += `${eol2}${repositories_without_database.length} repositories did not have a CodeQL database available:${eol}${repositories_without_database.join(', ')}`;
       logMessage += `${eol}For each public repository that has not yet been added to the database service, we will try to create a database next time the store is updated.`;
     }
-    if (response.errors.private_repositories?.length) {
-      logMessage += `${eol2}Non-public repositories:${eol}${response.errors.private_repositories.join(', ')}`;
+    if (private_repositories?.length) {
+      logMessage += `${eol2}${private_repositories.length} repositories are not public:${eol}${private_repositories.join(', ')}`;
```

Comment:
Since we're building a log message from:
- two end of lines
- a number
- a message
- another eol
- a list
in three separate situations, extracting those lines into a method might improve readability. 

e.g. 
```
logMessage += buildCutOffMessage(private_repositories, "repositories are not public");

func buildCutOffMessage(list, message) => `${eol2}${list.length} ${message}:${eol}${list.join(', ')}`; 
```